.github/workflows/build_and_test.yml

```diff
@@ -57,6 +57,13 @@ jobs:
         options: --health-cmd pg_isready --health-interval 5s --health-timeout 5s --health-retries 8
 
     steps:
+    - name: Bump max open files
+      run: |
+          sudo echo 'kong soft nofile 65536' | sudo tee -a /etc/security/limits.conf
```

Comment:
It might be better to create a file in /etc/security/limits.d/ (with the name kong.conf) and put these configurations into the file.

Reply:
done